lib/fizzy/parser.cpp

```diff
@@ -6,6 +6,11 @@
 
 namespace fizzy
 {
+inline std::string to_string(bytes_view b)
```

Comment:
After #204 is merged do we still want this helper? I think it looks nice, but don't care too much.